app/models/archived_petition.rb

```diff
@@ -0,0 +1,40 @@
+class ArchivedPetition < ActiveRecord::Base
+  OPEN_STATE = 'open'
```

Comment:
The `State` concern has many states that aren't applicable to archived petitions - using it seemed to complicate things rather than making them simpler.